src/main/java/core/basesyntax/service/impl/FruitShopServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.operation.OperationHandler;
+import java.util.List;
+
+public class FruitShopServiceImpl implements FruitShopService {
```

Comment:
inject OperationStrategy using a constructor. It can be the same for each method call.